### PR TITLE
chore(wip): use bookworm-slim for docker base image

### PR DESCRIPTION
**Pull request by owlas**
current issues:
- [ ] openssl not available? https://github.com/nodejs/docker-node/issues/1919
- [x] `.git` error from husky (no git?)

test-frontend test-backend test-cli

### CODE REVIEW

**Comment by rephus** [comment]
running tests

**Review by rephus — CHANGES_REQUESTED** [review]
We also need to update dockerfile-prs file

**Comment by rephus** [comment]
I've manually created a new project snowflake + github 

https://lightdash-pr-8648.onrender.com/projects/a646fe0e-220b-4a10-9d88-59d06fabdd30/tables/payments?

**Comment by rephus** [comment]
I got this error when trying to create a project 

image

I installed git-all from apt

**Comment by rephus** [comment]
Fixed the git issue

We also need to install git on stage 3 

image

File: dockerfile-prs [hunk 1]
```diff
@@ -16,12 +16,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
...
     unzip \
     wget \
+    git \ 
     && apt-get clean
 
 # Installing multiple versions of dbt
 # dbt 1.4 is the default
-RUN python3 -m venv /usr/local/dbt1.4
-RUN /usr/local/dbt1.4/bin/pip install \
+RUN python3 -m venv /usr/local/dbt1.4 \
```
Comment (rephus) [inline dockerfile-prs:24]: copy how we do it on `dockerfile`

**Comment by owlas** [comment]
Works well with bigquery + CLI workflow

**Review by rephus — APPROVED** [review]

**Pull request merged**